website/docs/core-concepts/stacks/yaml-functions/include.mdx

```diff
@@ -170,3 +170,9 @@ On macOS and Linux:
     values: !include './values.yaml "<yq-expression-with-spaces>"'
     description: !include '"component description.md"'
 ```
+
+Sometimes for more complex cases you may need to utilize 3x double quotes, for examples:
+```yaml
+vars:
+  my_array: !include '"https://foo.com/bar.json" """.some_key.array_one + .some_key.hyphenated-field.array_two"""'
```

Comment:
Please try this instead 
```suggestion
vars:
  my_array: !include >
    https://foo.com/bar.json
    ".some_key.array_one + .some_key.hyphenated-field.array_two"
```

Reply:
Doesn't work:
```console
ERRO invalid number of arguments in the Atmos YAML function: !include https://foo.com/bar.json .some_key.array_one + .some_key.hyphenated-field.array_two. The function accepts 1 or 2 arguments
```

Reply:
Based on what you share, it looks like the quote characters are stripped away in the folded multi line string. That is not the result of YAML but something that Atmos must be doing. Using the folded style, tells the YAML interpreter to treat the rest of the content literally, without evaluation.

This is why I am perplexed as to why I don't see quotes in your output.